google/cloud/security/inventory/inventory_loader.py

```diff
@@ -372,45 +242,70 @@ def _send_email(cycle_time, cycle_timestamp, status, pipelines,
         LOGGER.error('Unable to send email that inventory snapshot completed.')
 
 
-def _configure_logging(configs):
+def _configure_logging(loglevel):
     """Configures the loglevel for all loggers."""
-    desc = configs.get('loglevel')
-    level = LOGLEVELS.setdefault(desc, 'info')
+    level = LOGLEVELS.setdefault(loglevel, 'info')
```

Comment:
Hmmm, I was just following the existing code pattern.  :)  Honestly, I was wondering about the entire usage of FLAGS.FlagValuesDict() that bundles the flags into a dictionary.  Unless we change that, we should stick with using it.